spec.md

```diff
@@ -44,6 +44,10 @@ These are the definitions for some of the terms used in this specification:
     - The files in the `.reuse/` directory in the root of the Project. This
       directory MUST contain only files relevant for the operation of the REUSE
       Tool.
+    - SPDX documents in their various formats as defined in the [SPDX
+      Specification, Clause
+      4.4](https://spdx.github.io/spdx-spec/conformance/#44-standard-data-format-requirements)
```

Comment:
Either change this to point to the 2.2(.1) version directly or update the other URL to point to the current spec version.

Doing the latter would probably be easier, given the new docs URL/solution of SPDX changed; and enough, since we explicitly call for the SPDX version in our own spec, and SPDX tends to be backwards compatible as much as possible.

Will provide a separate PR to update the other links to the 2.2.1 version.

Reply:
Added my changes.

Reply:
On my end, this directs to 2.2.1, and is the same URL pattern as the ones you updated. Or do I miss something?

Reply:
Great, thanks! Could you approve the PR then please?